### PR TITLE
Whiteboard and Comment page layout updated

### PR DESCRIPTION
**Pull request by ritiksaxena124**
**Description**
1. This PR updates the layout of the whiteboarding page and commenting page
2. Some content is also updated on the whiteboarding page

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit af6b390c71c8be09a81d2a0ec76cfa419b490e98 at: https://6575c74840c3bb27c6950a1e--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit aef0b4925334a05f9ed084a75514b8fb0fdcc64a at: https://6575cc75b3088f2b385fdf25--layer5.netlify.app

File: src/sections/FeatureHero/index.js [hunk 1]
```diff
@@ -11,7 +9,7 @@ const FeatureHero = (props) => {
 
     const handleScroll = () => {
       const scrollPosition = window.scrollY;
-      const translateY = -scrollPosition / 5;
+      const translateY = -scrollPosition / 16;
```
Comment (leecalcote) [inline src/sections/FeatureHero/index.js:12]: A bit o' javascript (not purely css). Ok.

**Comment by leecalcote** [comment]
@ritiksaxena124 will you review these merge conflicts?

**Comment by ritiksaxena124** [comment]
> @ritiksaxena124 will you review these merge conflicts?

Sure 👍

**Comment by l5io** [comment]
🚀 Preview for commit befed7dc810b24d2d5af8abf50bb86e64a341b68 at: https://657c9fe71d85b81c6d439468--layer5.netlify.app

**Comment by leecalcote** [comment]
Some great work here.

**Pull request merged**